site/content/deployment/openshift.md

```diff
@@ -10,10 +10,35 @@ to communicate with each other.
 
 ## Prerequisites
 
-The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster just
-using the OpenShift client tools that can be downloaded from the [OpenShift Origin](https://github.com/openshift/origin/releases) project repository.
+The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster choosing
+between using :
+
```

Comment:
When you use minishift it installs oc client tools for you so you don't need the oc client tools downloaded as required in the first approach.